tests/test_ontology.py

```diff
@@ -29,10 +26,13 @@ def test_nbadapter_repo_annotations(test_notebook_repo):
 
     logger.info(G.serialize(format="turtle"))
     
-    assert (oda["emin_keV"], a, oda["emin"]) in G
-    assert (oda["emin_keV"], a, oda["keV"]) in G
+    # note that here, oda:keV is understood as "something that is expressed in keV"
+    # this is different from the unit:keV .
```

Comment:
Not strictly necessary, but probably good to have an ability to add smth like `oda:unit unit:keV`
Then `unit:` prefix should be declared in extra_ttl.

In current testcase ontology there are no extra prefixes, oda:keV is instance of Unit, and "something that is expressed in keV" is called oda:par_keV

Reply:
> Not strictly necessary, but probably good to have an ability to add smth like `oda:unit unit:keV` Then `unit:` prefix should be declared in extra_ttl.

Ok, I add a test. But we probably should not derive here that class with annotation of `oda:unit unit:keV` is sameAs `oda:keV`. We can do it in dispatcher, right? Where similar transformations are done.

> In current testcase ontology there are no extra prefixes, oda:keV is instance of Unit, and "something that is expressed in keV" is called oda:par_keV

I see, but I think it's more intuitive for workflow developers to say `pivot_energy=30 # oda:keV` than `pivot_energy=30 # oda:par_keV`.
It's a property of expression in human language. E.g. "apple is sweet" (which appears to be assertion of being) is used for more fluid expression instead of "apple has taste of sweet" (which is also valid of course).
So if we could keep the expressions as concise as possible they will be better understood.

Reply:
It's not too complex to change in your proposed ontology?

Reply:
> But we probably should not derive here that class with annotation of `oda:unit unit:keV` is sameAs `oda:keV`. We can do it in dispatcher, right?

Of course, it should be returned as-is. I'm not sure about inferring sameAs relation in dispatcher, but both ttl can be used and will lead to same result.
I wanted to keep both ways of annotating to allow using units/formats from external ontologies (for which oda:smth is not defined) in the future. 

> It's not too complex to change in your proposed ontology?

I haven't found the dedicated functionality in Protege, but there are not much of changes, I will do it.